### DISCUSSION TITLE
Recent added

### DISCUSSION THREAD

**Discussion by godlike73** [advplyr/audiobookshelf#1001] [Q&A]
Is there a way to select the amount of books in recently added?  Sometimes if adding too many at a time makes some not show up?

**Comment by advplyr** [comment]
The limit is 10 items per shelf for most shelves. There are plans to have some form of a "view all" button that will show all items. I'm not sure how this will work with recently added, probably just show the most recent 50 or something like that.

**Comment by skyzuma** [comment]
"view all" button ... better option is to make "recently added" linkable / clickable to library tab with sorting: date (latest)

same with "recent series" > linkable / clickable to library tab with sorting: series(all / latest) with selected "collapse series" or any similar

maybe same with "listen again"

newest authors  linkable / clickable to authors page with an sorting system
